pwa/src/components/expandableLeadParagraph/ExpandableLeadParagraph.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import * as styles from "./ExpandableLeadParagraph.module.css";
+import { Button, Paragraph } from "@utrecht/component-library-react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
```

Comment:
We usually use `faChevronRight` for collapsibles so when the collapsible is closed the chevron faces to the right and when we open the collapsible we rotate it `90deg` so it faces downwards.